laborator/lab-05/README.md

```diff
@@ -0,0 +1,313 @@
+
+### Subsecțiuni ale registrelor
+
+În anumite cazuri dorim să modificăm valori ce sunt reprezentate pe mai puțin de 4 octeți (spre exemplu, lucrul cu șiruri de caractere). Pentru aceste situații, procesoarele x86 ne oferă posibilitatea de a lucra cu subsectiuni de 1, respectiv 2 octeți ale registrelor EAX, EBX, ECX, EDX.
+
+În imaginea de mai jos sunt reprezentate registrele, subregistrele și dimensiunile lor.
+
+![Registre x86_32](https://www.tenouk.com/Bufferoverflowc/Bufferoverflow1_files/image005.png)
+
+> **WARNING**
+> Subregistrele fac parte din registre, ceea ce înseamnă că dacă modificăm un registru, în mod implicit modificăm și valoarea subregistrului.
+
+> **NOTE**
+> Subregistrele se folosesc în mod identic cu registrele, doar că dimensiunea valorii reținute este diferită.
+
+> **NOTE**
+> Pe lângă registrele de bază mai există și șase registre segment corespunzătoare unor anumite zone după cum se observă în imagine:
+> ![Registre Segment](https://ocw.cs.pub.ro/courses/_media/iocla/laboratoare/six_segments.png)
+
+### Declarări statice de regiuni de memorie
+
+Declarările statice de memorie (analoage declarării variabilelor globale), în lumea x86, se fac prin intermediul unor directive de asamblare speciale. Aceste declarări se fac în secțiunea de date (regiunea .DATA). Porțiunilor de memorie declarate le pot fi atașate un nume prin intermediul unui label pentru a putea fi referite ușor mai târziu în program. Urmăriți exemplul de mai jos:
+
+```Assembly
+.DATA
+    var        DB 64    ; Declară un octet conținând valoarea 64. Etichetează
+                        ; locația de memorie cu "var".
+    var2       DB ?     ; Declară un octet neinițializat etichetat cu "var2".
+               DB 10    ; Declară un octet neetichetat, inițializat cu 10. Acest
+                        ; octet va fi plasat la adresa (var2 + 1).
+    X          DW ?     ; Declară un cuvânt(2 octeți) neinițializat, etichetat cu "X".
+    Y          DD 3000  ; Declară un cuvânt dublu (4 octeți) cu eticheta "Y",
+                        ; inițializat cu valoarea 3000.
+    Z          DD 1,2,3 ; Declară 3 cuvinte duble (a câte 4 octeți fiecare)
+                        ; incepând cu adresa "Z" și inițializate cu 1, 2, respectiv 3.
+                        ; De exemplu, 3 va fi plasat la adresa (Z + 8).
+```
+
+> **NOTE**
+> DB, DW, DD sunt directive folosite pentru a specifica dimensiunea porțiunii:
+> Directivă | Rol | Dimensiune
+> ----------|-----|-----------
+> DB | Define Byte | 1 octet (8 biți)
+> DW | Define Word | 2 octeți (16 biți)
+> DD | Define Double Word | 4 octeți (32 biți)
```

Comment:
Nu este generat corect tabelul. Dacă nu funcționează tabelul în acest tip de citări, încearcă să îl pui separat și faci referire la el. Dar cel mai bine ar fi dacă se poate genera corect așa cum este acum.

Reply:
Nu sunt sigur de ce nu este afișat corect pe GitHub; am un preview direct in editor și acolo arată corect

Reply:
Github are un format special, puțin modificat de markdown (sunt foarte multe variante de markdown). Poți încerca să modifici direct din interfața web, dar să fii atent cu ce mesaj faci commit.